test/functional/test_framework/test_node.py

```diff
@@ -17,9 +17,12 @@
     assert_equal,
     get_rpc_proxy,
     rpc_url,
+    wait_until,
 )
 from .authproxy import JSONRPCException
 
+BITCOIND_PROC_WAIT_TIMEOUT = 60
```

Comment:
This constant is now only used in one place. Perhaps remove it as a constant and add it as an integer literal in the definition of `wait_until_stopped()` with a comment.

If not, perhaps move this constant assignment down to just above the call to `wait_until_stopped()`

Reply:
I prefer to put constants at the top of a file and my personal taste is that it is ugly to place them between or inside methods/functions.

Reply:
That's reasonable